packages/lib-user/dev/components/App/App.js

```diff
@@ -0,0 +1,134 @@
+import oauth from 'panoptes-client/lib/oauth.js'
+import { useEffect, useState } from 'react'
+
+import { usePanoptesUser } from '@hooks/index.js'
+import { GroupStats, UserStats } from '@components/index.js'
+
+function App ({
+  groups = null,
+  users = null
+}) {
+  const [loading, setLoading] = useState(false)
+  const [userAuth, setUserAuth] = useState(null)
+
+  const { data: user, error, isLoading: userLoading } = usePanoptesUser(oauth)
+
+  useEffect(() => {
+    async function initAuthorization () {
+      setLoading(true)
+
+      try {
+        const userAuth = await oauth.init('357ac7e0e17f6d9b05587477ca98fdb69d70181e674be8e20142e1df97a84d2d');
```

Comment:
I noticed this is used in lib-classifier too, but where does this string come from? Just for my own understanding...

Reply:
That’s the OAuth client ID from Doorkeeper. TBH, each app should generate its own key, rather than reusing keys.

Reply:
I thought it might be the dev classifier ID, but it's not. 

https://panoptes-staging.zooniverse.org/oauth/applications/47

Reply:
My bad. This is the 'lib-user dev' app ID. 

https://panoptes-staging.zooniverse.org/oauth/applications/63